templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+                        </div>
+                    </div>
+                </article>
+            </div>
+        </div>
+	</div>
+	<div class="container" style="display: flex; justify-content: flex-end; padding-top: 72px;">
+        <div class="column is-half">
+            <p style="font-family: Cabin; family-style: normal; font-size: 28px; color: #000000;">BoilerMake is Purdue University's largest hackathon. Over the past seven years,
+             thousands of students from across the country have come to BoilerMake. This event is for experienced hackers, newbies to technology, and everyone in between.</p>
+        </div>
+    </div>
+    <div class="container" style="display: flex; justify-content: space-between;">
+        <div class="column is-half">
+            <img alt="Blue van" src={{ static_path "bmviii_van.svg" }}>
+        </div>
+        <div class="column is-half"; style="display: flex; justify-content: space-between; padding-top: 120px;">
+            <div class="column is-one-third">
+                <img src={{ static_path "bmvii_why_build.svg" }} class="bmvii-why__img" alt="Gears Turning">
+                <p class="subtitle is-5 has-text-white has-text-weight-bold">Build Creative Projects</p>
+                <img src={{ static_path "bmvii_why_resume.svg" }} class="bmvii-why__img" alt="Good resume">
+                <p class="subtitle is-5 has-text-white has-text-weight-bold">Build Creative Projects</p>
+            </div>
+            <div class="column is-one-third">
+                <img src={{ static_path "bmvii_why_companies.svg" }} class="bmvii-why__img" alt="Formal Attire">
+                <p class="subtitle is-5 has-text-white has-text-weight-bold">Talk to Company Reps</p>
+                <img src={{ static_path "bmvii_why_skills.svg" }} class="bmvii-why__img" alt="Brain">
+                <p class="subtitle is-5 has-text-white has-text-weight-bold">Learn New Skills</p>
+            </div>
+            <div class="column is-one-third">
+                <img src={{ static_path "bmvii_why_memories.svg" }} class="bmvii-why__img" alt="Floppy Disk">
+                <p class="subtitle is-5 has-text-white has-text-weight-bold">Make Incredible Memories</p>
+                <img src={{ static_path "bmvii_why_prizes.svg" }} class="bmvii-why__img" alt="Trophy">
+                <p class="subtitle is-5 has-text-white has-text-weight-bold">Win Cool Prizes</p>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="section has-background-image" id="surfboards" style='background-image: url({{ static_path "bmviii_surfboard_background.svg" }})'>
+    <div style="display:flex; justify-content: center align-self: flex-end; padding-bottom:0">
+        <img src={{ static_path "bmviii_surfboards.svg" }} class="bmvii-why__quarry" alt="Surfboards">
+    </div>
+</section>
+
+<section class="section has-background-image" id="faq" style='background-image: url({{ static_path "bmviii_faq.svg" }})'>
```

Comment:
I think expanding a question shouldn't show through to the background, it should have just like a white background so you can read the text easier
![Screen Shot 2020-10-05 at 21 21 26](https://user-images.githubusercontent.com/10781907/95148230-6f395480-0748-11eb-90d3-46887ee0b74f.png)